### PR TITLE
Make toggle-split switch to the students tab

### PR DESCRIPTION
**Pull request by chloelee767**
### What this does
This PR changes the behaviour of `toggle-split` so that it always switches to the students tab, which is probably what most users would expect (see #144). This behaviour is also now documented in the UG to avoid any confusion.

### How to test
While the students and rooms tabs are separate:
Run `toggle-split` from the students tab, rooms tab and bin tab. Verify that the combined tab is always shown after running the command.

While the students and rooms tabs are combined:
Run `toggle-split` from the combined tab and the bin tab. Verify that the students tab is always shown after running the command.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -443,13 +443,17 @@ Format: `exit`
 #### View students and rooms side by side: `toggle-split`
 While allocating rooms to students, it is probably easier to view rooms and students at the same time. If the rooms and students tabs are currently separate, `toggle-split` will merge the students and rooms tab into 1 tab that shows them side by side, which is shown the image below. You can use the `rooms` or `students` commands to switch to the combined tab as usual.
 
+Note that `toggle-split` will cause the application to switch to the combined students and rooms tab, regardless of what tab you previously had open.
```
Comment (JingYenLoh) [inline docs/UserGuide.md]: Perhaps these notes can be wrapped in an info modal to stand out, similar to the ones used in certain sections of the UG.

image

Comment (chloelee767) [reply docs/UserGuide.md]: Oh that's a good suggestion, will do!

**Review by Lysire — APPROVED** [review]
LGTM!

**Review by JingYenLoh — APPROVED** [review]
Manually tested, LGTM!

**Pull request merged**